tf_files/aws/csoc_common_logging/root.tf

```diff
@@ -13,6 +13,9 @@ module "logging" {
   csoc_account_id  = "${var.csoc_account_id}"
   threshold        = "${var.threshold}"
   slack_webhook    = "${var.slack_webhook}"
```

Comment:
does this work with `terraform12` ?  maybe do something like this:
* add an optional `gen3Metadata.json` to each `tform/aws/script/` folder 
* modify `gen3 workon` to check if that file exists, and set `GEN3_TERRAFORM_PATH` or whatever to point at terraform 12 if it supports terraform 12
* modify `terraform.sh` to key off the TERRAFORM variable set by `gen3 workon` ...

Reply:
I would just dedicate a single JIRA/sprint to make this work. Not as trivial as it sounds.